Examples/transformers-cli/Sources/transformers-cli/Transformers.swift

```diff
@@ -0,0 +1,141 @@
+import ArgumentParser
+import CoreML
+import Foundation
+import Generation
+import Models
+
+@available(macOS 15.0, iOS 18.0, *)
```

Comment:
Do we need this annotation if the package already defines these supported platform?

Reply:
Actually, I ended up not imposing the restriction on the package itself (i.e., preserve what's currently in main), but restrict Core ML features with availability annotations.